wrappers/dotnet/indy-sdk-dotnet/WalletApi/Wallet.cs

```diff
@@ -13,12 +14,13 @@ namespace Hyperledger.Indy.WalletApi
     /// Represents a wallet that stores key value records and provides static methods for managing
     /// wallets.
     /// </summary>
+    [SuppressMessage("ReSharper", "InconsistentNaming")]
     public sealed class Wallet : IDisposable
```

Comment:
small thing, I feel this code analysis tag for resharper should be removed.  not everyone uses resharper.  unless the intent is to make resharper a requirement for this project.